flight/Libraries/uavtalk.c

```diff
@@ -254,6 +261,26 @@ UAVTalkRxState UAVTalkProcessInputStreamQuiet(UAVTalkConnection connectionHandle
 		if (iproc->rxCount < 4)
 			break;
 
+		if (iproc->type == UAVTALK_TYPE_FILEREQ) {
```

Comment:
Yes, the plan is to allow pushing files the other way, though the state machine of the initiator is slightly trickier.